torch/csrc/jit/custom_operator.h

```diff
@@ -182,26 +182,6 @@ Operator createOperator(
 
   auto schema = torch::jit::detail::inferAndCheckSchema<Traits>(schemaOrName);
 
-  // [custom operator aliasing] Currently, we have no way for the user to
```

Comment:
Why did you delete this ?

Reply:
#18589 makes it unnecessary

Reply:
I think we should do this clean up in a separate PR where you track down all references to this note and fix those up as well. Don't leave the pointers dangling!